ruby/rubocop/config/base.yml

```
@@ -231,12 +231,15 @@ Metrics/AbcSize:
 Metrics/BlockLength:
   Enabled: true
   CountComments: false
-  IgnoredMethods:
-    - class_methods
-    - concern
-    - describe
-    - included
-    - refine
+  AllowedMethods:
+    - Enabled
+    - CountComments
+    - Max
+    - CountAsOne
+    - AllowedMethods
+    - AllowedPatterns
+    - Exclude
+    - inherit_mode
```

`AllowedMethods` に「何を渡すべきなのか」をちょっと勘違いしてそうですね。

「期待している挙動」を変えずに「warning が出ないようにしたい」のであれば、↓こうなるはず。
```suggestion
  AllowedMethods:
    - class_methods
    - concern
    - describe
    - included
    - refine
```

---
期待する挙動は以下のとおりです。  
（書いてて「ホンマに？」と思ってしまったけど、とりあえず今回のPRでは「warning が出ないこと」を目標にしましょう）

```ruby
RSpec.describe Target do
  # これは指摘されない - `AllowedMethods` に含まれてるから
  describe "#call" do
    # 26行以上のコード
  end

  # これは指摘される - `AllowedMethods` に含まれてないから
  context "some context" do
    # 26行以上のコード
  end
end
```

---
RuboCop 本体が提供している「デフォルト設定」も参考にしてくださいな。
https://github.com/rubocop/rubocop/blob/3877dd044cc7d83cb2d107da99bea3b968f9a40e/config/default.yml#L2517-L2531

---

完全に勘違いしてました...
実際に、26行以上のコード書いてやってみたら確かにcontextの方だけwarningなること確認しました

```bash
test_spec.rb:71:3: C: Metrics/BlockLength: Block has too many lines. [60/25]
  context 'some context' do ... # AllowedMethodsに書いていないと確かにwarningになる
  ^^^^^^^^^^^^^^^^^^^^^^^^^
```